internal/constants.go

```diff
@@ -0,0 +1,33 @@
+package internal
+
+var PARENS = []string{"(", ")"}
+
+var BRACES = []string{"{", "}"}
+
+var SINGLECHAROPERATORS = []string{"+", "-", "*", ".", ",", ";"}
```

Comment:
```suggestion
var SINGLE_CHAR_OPERATORS = []string{"+", "-", "*", ".", ",", ";"}
```

If we're avoiding underscores because of some kind of Go lint rule, we should just disable that - seems like a hindrance to readability

Reply:
Yep, done.